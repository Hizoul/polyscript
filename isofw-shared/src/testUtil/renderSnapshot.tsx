import * as React from "react"
const render = require("preact-render-to-json")

const renderSnapshot = (component: any, msg: string) => {
  expect(render(component)).toMatchSnapshot(msg)
}

export default renderSnapshot
