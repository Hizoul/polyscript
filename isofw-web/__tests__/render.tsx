import renderSnapshot from "isofw-shared/src/testUtil/renderSnapshot"
import Comp from "isofw-web/src/comp"
import * as React from "react"

test("Verify Component", () => {
  renderSnapshot(<Comp />, "rendered")
})
