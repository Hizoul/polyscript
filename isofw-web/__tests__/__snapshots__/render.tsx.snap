// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Verify Component: rendered 1`] = `
<div>
  <span>
    Thisisweb
  </span>
  <img
    source="fileMock"
    src="fileMock"
    style={
      Object {
        "height": 100,
        "width": 100,
      }
    }
  />
</div>
`;
